mautic/mautic
# Prevented plugin injected assets from getting inserted into builder preview HTML

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? |  y
| New feature? |  y
| Related user documentation PR URL | na
| Related developer documentation PR URL |  todo
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

When a plugin injects custom assets via the assets helper, the assets are injected into the head of the builder preview iframe (email/page) which contaminates the theme's CSS. So this PR fixes that by defining a context for which assets are to be applied allowing a plugin to specifically inject css into the builder preview or just the main app.

#### Steps to test this PR:
1. Ensure the builder and overall UI still function correctly.
2. Run new unit test
